com/williamfiset/algorithms/datastructures/linkedlist/DoublyLinkedList.java

```diff
@@ -77,6 +77,38 @@ public void addFirst(T elem) {
     size++;
   }
 
+  // Add an element at a specified index
+  public void addAt(int index, T data) throws Exception
```

Comment:
Please add tests to the [LinkedListTest](https://github.com/williamfiset/Algorithms/blob/master/javatests/com/williamfiset/algorithms/datastructures/linkedlist/LinkedListTest.java) file to verify the functionality of this new function :) I have more instructions in the [wiki](https://github.com/williamfiset/Algorithms/wiki)

Reply:
Hi Prashana, please move the changes in pull request #102 to this pull request so that all the changes are in one place.

Reply:
Hi william, I have moved the changes here.

Reply:
Thanks for the changes!